knowledge-base/old-webforms-viewer-cannot-display-list-items.md

```diff
@@ -0,0 +1,41 @@
+---
+title: The Old ASP.NET Webforms ReportViewer Cannot Display Table, List, or Crosstab Item Reports (IE)
+page_title: The Old ASP.NET Webforms ReportViewer Cannot Display Table, List, or Crosstab Item Reports (IE)
+description: "Learn how to handle the situation when the old ASP.NET Webforms ReportViewer does not display reports holding Table, List, or Crosstab items."
+slug: old-webforms-viewer-cannot-display-list-items
+tags: telerik, reporting, troubleshoot, asp, net, webforms, old, reportviewer, doesn't, display, list, table, crosstab, item, report
+published: True
+type: troubleshooting
+res_type: kb
+---
+
+## Environment
+
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+		<tr>
+			<td>Version</td>
+			<td>Q2 2012 and newer</td>
+		</tr>
+	        <tr>
+			<td>Report Viewers</td>
+			<td>WinForms, WPF, ASP.NET</td>
```

Comment:
Related only to the legacy AAP.NET WebForms Viewer

Include a note about this so that it is clear to the readers. Most of them don't use this viewer anymore and we recommend them to migrate to the Html5 Viewer